.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         ia2-tracer: [ON, OFF]
     steps:
       - name: Setup Rust
-        uses: actions-rust-lang/setup-rust-toolchain@v1
+        run: rustup toolchain install stable-x86_64-unknown-linux-gnu
```

Comment:
Does it make sense to do this same thing for the aarch64 one?

Reply:
It doesn't really matter as that test can run in parallel with the ones on our runner.

Reply:
Yeah but it's helpful to be consistent.